utils/config.go

```diff
@@ -470,7 +470,7 @@ func getClientConfig(c *model.Config) map[string]string {
 			props["EnableCluster"] = strconv.FormatBool(*c.ClusterSettings.Enable)
 		}
 
-		if *License.Features.Cluster {
+		if *License.Features.Metrics {
```

Comment:
I'm not sure this is a mistake, I don't think the license file has a metrics switch yet. Even if it was I'd rather not change this so close to release